Oleksii_Fedorchuk/HW16_Oleksii_Fedorchuk/task1.py

```diff
@@ -0,0 +1,31 @@
+class CustomIterator:
+    def __init__(self, sequence: list, start_index: int, end_index: int):
+        self.__sequence = sequence
+        self.__start_index = start_index
+        self.__end_index = end_index
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.__start_index == 0:
+            raise IndexError("The start index should be more then 0!")
```

Comment:
why more than 0?

Reply:
0 is a valid value actually because indexing starts from 0 in python

Reply:
Fixed.